### PR TITLE
[DEV-10834] Increase the nested_object.limit value for the covid19-faba index [hotfix][qat]

### PR DESCRIPTION
**Pull request by aguest-kc**
Duplicate of #4062 for QAT

### CODE REVIEW

**Comment by aguest-kc** [comment]
🛑 Don't merge QAT into the new branch until the Staging and Master PRs have been merged. 🛑

**Review by collinwr — APPROVED** [review]

**Pull request merged**
